src/Compilers/CSharp/Portable/Parser/LanguageParser_InterpolatedString.cs

```diff
@@ -147,46 +134,76 @@ private ExpressionSyntax ParseInterpolatedStringToken()
             return CheckFeatureAvailability(result, MessageID.IDS_FeatureInterpolatedStrings);
         }
 
+        private static InterpolationSyntax ParseInterpolation(CSharpParseOptions options, string text, Lexer.Interpolation interpolation, bool isVerbatim)
```

Comment:
made static so it's clear that you can't call instance methods both on the original 'parser' instance and the 'tempParser' created within.